opencti-platform/opencti-graphql/src/database/xtm-obas.ts

```diff
@@ -11,12 +11,13 @@ import { ENTITY_TYPE_CONTAINER_CASE_RFT } from '../modules/case/case-rft/case-rf
 import { ENTITY_TYPE_THREAT_ACTOR_INDIVIDUAL } from '../modules/threatActorIndividual/threatActorIndividual-types';
 
 const XTM_OPENBAS_URL = conf.get('xtm:openbas_url');
+const XTM_OPENBAS_API_OVERRIDE_URL = conf.get('xtm:openbas_api_override_url');
```

Comment:
```suggestion
const XTM_OPENBAS_API_OVERRIDE_URL = conf.get('xtm:openbas_api_url');
```
I think that override is not needed in the variable name.